### PR TITLE
Adds icons for all achievements and adds 3 new achievements

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus & Fikou for 90% of the icons
 add: New achievements, go find out what they are yourself
 fix: achievements now unlock properly
 imageadd: New icons for all achievements
/:cl:

Rule 8 achievement can be unlocked by doing some shenanigans with plushies. Longshift PR is just what happens in most rev rounds, and mining skill is just finishing the hardest challenge for miners

### CODE REVIEW

**Comment by Cyprex** [comment]
Nothing shows up on live servers when I open the achievements menu on IE 8/wine, is that intended?

**Comment by Qustinnus** [comment]
No, that shouldn't happen @Cyprex . Is it only at roundstart? Does this happen to other UI?

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -170,10 +170,16 @@
...
+	var/speed_round = FALSE
+	if(world.time - SSticker.round_start_time <= 90 SECONDS)
+		speed_round = TRUE
+
 	for(var/client/C in GLOB.clients)
 		if(!C.credits)
 			C.RollCredits()
 		C.playtitlemusic(40)
+		if(speed_round)
+			C.give_award(/datum/award/achievement/misc/speed_round, user)
```
Comment (Time-Green) [inline code/__HELPERS/roundend.dm]: Is this really a good idea? This is going to encourage revheads to instantly suicide to get achievements

Comment (Qustinnus) [reply code/__HELPERS/roundend.dm]: I can raise the number a bit, I doubt people will be that autistic but nothing surprised me by now either.

Comment (PKPenguin321) [reply code/__HELPERS/roundend.dm]: I would raise it to like 5 minutes at least

Comment (cacogen) [reply code/__HELPERS/roundend.dm]: If revheads instantly suicide that's an administrative issue, pretty egregious thing to do that would probably result in being banned from the role

Comment (ATH1909) [reply code/__HELPERS/roundend.dm]: make this only give out the medal to greentexting antags (NOT to ordinary crewmembers or to redtexting antags)

File: code/datums/achievements/misc_achievements.dm [hunk 2]
```diff
@@ -1,42 +1,63 @@
...
+
+/datum/award/achievement/misc/rule8
+	name = "Rule 8"
+	desc = "Call an admin this is ILLEGAL!!"
+	hub_id = MEDAL_RULE8
+	icon = "rule8"
+
+/datum/award/achievement/misc/speed_round
+	name = "Long shift"
+	desc = "Well, that didn't take long"
```
Comment (ATH1909) [inline code/datums/achievements/misc_achievements.dm]: add a period after "long"

**Comment by actioninja** [comment]
conflict

**Comment by Qustinnus** [comment]
not according to gitkraken

**Comment by Fikou** [comment]
image
ill just plug this here

**Comment by Qustinnus** [comment]
sorry buddy only shadowlight has the medal

**Comment by Qustinnus** [comment]
Someone might wanna still finish this so fikou doesn't have a stroke

**Pull request closed without merging**